src/main/resources/files/mp3_x/index.html.tpl

```diff
@@ -45,6 +45,7 @@
 
 [# th:if="${mp_open_tracing}"]
 <h3>Open Tracing</h3>
+If you have <pre>./jaeger-all-in-one</pre> running, open <a href="http://localhost:16686/">http://localhost:16686</a>
```

Comment:
This isn't true for all implementations as Jaeger isn't a default/required supported Tracer within MP

Reply:
@rdebusscher Hmm, are you saying that even if a runtime correctly passes Open Tracing TCK, it might not work with Jaeger?  Or are you having an issue with advocating/suggesting/promoting Jaeger per se?

Reply:
Yes, Payara Micro passes TCK without Jaeger or Zipkin.

Tracing spec doesn't say anything about an 'implementation' behind it.

Reply:
@rdebusscher I kinda sneaked this in, because I used it during presentations and found it convenient. If you would like it removed, what would you suggest? We don't have any means of running that example easily for a new/learning user.

Reply:
>I kinda sneaked this in

Never a good idea ...

I thought there was an issue for Tracing example, but can't find it anymore.

For Payara Micro, the internal tracer doesn't need any config. For Jaeger or Zipkin you need additional config and jar files. That is probably also the case for other implementations.

So it is not easy to have such thing in the starter which should work out of the box.

Reply:
@rdebusscher O.K., now I understand. Jaeger works without additional jars with Thorntail and Quarkus, but one would need some additional jars to make it work with Payara Micro, right?

If that is the case, I suggest moving this to server - specific index.html.   If you note the Quarkus README, I am describing the Jaeger there too...

See [readme.md.tpl](https://github.com/eclipse/microprofile-starter/blob/39d03afb071d051b62ec37e97b091c44a74670c4/src/main/resources/files/quarkus/readme.md.tpl#L104)

Reply:
@rdebusscher 

The latest commit does this:

Quarkus shows:

```
Open Tracing
If you have

./jaeger-all-in-one

running, open http://localhost:16686
```

while Payara and others show:

```
Open Tracing
If you have a tracer running, you can open it now.
```

Is this acceptable?